kokoro/presubmit/presubmit.sh

```diff
@@ -50,17 +50,36 @@ function check() {
   echo "${green}OK${normal}"
 }
 
+function run_copyright_headers() {
+  tmpfile=`mktemp`
+  for suffix in "cc" "cpp" "frag" "glsl" "go" "h" "hpp" "java" "js" "sh" "vert" "xml"; do
+    # Grep flag '-L' print files that DO NOT match the copyright regex
+    # Filter out third party files
+    # Grep seems to match "(standard input)", filter this out in the for loop output
+    find . -type f -name "*.${suffix}" | grep -v "./tools/build/third_party" | xargs grep -L "Copyright .* Google"
+  done | grep -v "(standard input)" > ${tmpfile}
+  if test -s ${tmpfile}; then
+    # tempfile is NOT empty
+    echo "Copyright issue in these files:"
+    cat ${tmpfile}
+    rm ${tmpfile}
+    return 1
```

Comment:
It's disappointing that this presubmit doesn't just fix the files in question, like all the others do.